server/mdm/assets/assets.go

```diff
@@ -101,25 +101,30 @@ func ABMToken(ctx context.Context, ds fleet.MDMAssetRetriever, abmOrgName string
 		return nil, fmt.Errorf("parsing ABM certificate: %w", err)
 	}
 
-	return DecryptRawABMToken(
-		assets[fleet.MDMAssetABMTokenDeprecated].Value, //abmTok.EncryptedToken,
+	oAuthTok, _, err := DecryptRawABMToken(
+		assets[fleet.MDMAssetABMTokenDeprecated].Value, // abmTok.EncryptedToken,
 		leaf,
 		assets[fleet.MDMAssetABMKey].Value,
 	)
+	if err != nil {
+		return nil, fmt.Errorf("decrypting ABM token: %w", err)
+	}
+
+	return oAuthTok, nil
 }
 
-func DecryptRawABMToken(tokenBytes []byte, cert *x509.Certificate, keyPEM []byte) (*nanodep_client.OAuth1Tokens, error) {
+func DecryptRawABMToken(tokenBytes []byte, cert *x509.Certificate, keyPEM []byte) (*nanodep_client.OAuth1Tokens, []byte, error) {
```

Comment:
I think it'd be helpful to name the returned values - the second one is the raw JSON of the token?

Reply:
I actually didn't need that new value at all, so I just returned it to the previous state.